src/updates/node/installer.ts

```diff
@@ -0,0 +1,38 @@
+import { exec } from 'child_process';
+import * as semver from 'semver';
+
+export function checkNodeInstalled(): boolean {
+	try {
+		exec('node -v');
+	} catch (e) {
+		return false;
+	}
+	return true;
+}
+
+export function checkNodeVersion(): string {
+	let version = '';
+	try {
+		exec('node --version', function (err, stdout) {
+			if (err) {
+				throw err;
+			}
+			if (!semver.satisfies(stdout, '>=10.13')) {
+				throw new Error('Titanium requires node 10.13 or greater.');
+			}
+			version = stdout;
+		});
+	} catch (e) {
+		throw new Error('Please install node to continue.');
+	}
+	return version;
+}
+
+export function getNodeVersionUrl(version: string): string {
```

Comment:
Maybe this could become the `installUpdate` function? I'm not sure how best we can respond back to the callee's to indicate they need to open the URL though